scripts/clean.sh

```diff
@@ -2,6 +2,12 @@ echo
 echo "=======================MobSF Clean Script for Unix======================="
 echo "Running this script will delete the Scan database, all files uploaded and generated."
 
+script_path=$(dirname $0)
+if [ "$script_path" != "." ] && [ "$script_path" != "scripts" ] && [ "$script_path" != "./scripts" ]; then
```

Comment:
Will this affect `setup.sh` ?
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/setup.sh#L21

Reply:
setup.sh will continue working  it's the case where script_path="scripts"